lib/persistedQueryDefaults.js

```diff
@@ -0,0 +1,32 @@
+const crypto = require('crypto')
+
+const PersistedQueryDefaults = {
+  Prepared: (persistedQueries) => ({
```

Comment:
I prefer to keep naming of those lowercase

Reply:
No worries